`/usr/bin/python ficheiro-saida.py` returned 0.
